.gitignore

```diff
@@ -0,0 +1,36 @@
+# See https://help.github.com/articles/ignoring-files/ for more about ignoring files.
```

Comment:
All errors should be named correctly, as in requirement "Kalkulator musi wyświetlać **odpowiednie** informacje o błędach arytmetycznych (np. dzielenie przez zero, mnożenie zbyt dużych liczb) w polu wyniku operacji (nie jako wyskakujące okno)."
![image](https://github.com/user-attachments/assets/a97de0f5-ce84-45ee-96e4-c42ead023077)

When there is division by zero app should show the user "Division by zero" or similar error, and when there is Too large numbers the error in the field should be different than from the on of division by zero